test/karma/test-app/slot-fallback/karma.spec.ts

```diff
@@ -1,7 +1,6 @@
 import { setupDomTests, waitForChanges } from '../util';
 
-// TODO(STENCIL-18) Restore this test and fix the underlying issue.
-xdescribe('slot-fallback', () => {
+describe('slot-fallback', () => {
```

Comment:
Went ahead and restored these tests while I was in here since I needed to add some new ones anyway.